### ISSUE TITLE
reactotron-react-native and reactotron-react-js v1.3.1 and v1.4.0 aren't published on npm

### ISSUE THREAD

**Issue by tsemerad** [reactotron/reactotron#244]
I'm in the process of setting up Reactotron on my project for the first time, and I'm trying to follow the redux setup instructions and it keeps telling me that `createStore is not a function`. It appears the issue is that I'm using `reactotron-redux@1.4.0`, but `reactotron-react-native@1.3.0` (which doesn't yet have the `createStore` function). Is it just an oversight that the latest versions of `reactotron-react-native` and `reactotron-react-js` haven't been published to npm?

**Comment by rmevans9** [comment]
the createStore function gets put onto the Reactotron object when you setup the redux plugin. I would suspect that your Reactotron setup is not occurring prior to you setting up your redux store.

**Comment by tsemerad** [comment]
Ok, thanks! I didn't realize that. I'll look into that.

**Comment by skellock** [comment]
Ya, like @rmevans9 said.  That `ReactotronConfig.js` file you create?  That needs all the plugins registered there via `use()` calls.  Once you `use()` a plugin, those functions like `createStore()` and others become available on the `Reactotron` object.

:)

As far as versions are concerned...  sorry ya, it's confusing.. here's the scoop:

I'm using [Lerna](https://github.com/lerna/lerna) to manage the multiple packages.  

They have 2 strategies for version numbers.  I picked the one that made the most sense at the time, but it's really confusing for humans when they look at it.

Only bug fixes & features that touch 1 of the 8 packages will bump that version.

I'm contemplating just bumping all version numbers of all things in lock step.  That will mean it's possible for a package to bump version without having anything change.

Which is goofy too.  

Basically, run `npm outdated` and if it says something is new, grab it unless I bump a major version (like go up to 2.x).

Haha, sorry for the confusion.

**Comment by skellock** [comment]
Sorry, i did a crap job of explaining that.

The version numbers bump only if code inside changes or a dependency changes.  In the case of `reactotron-react-native` and `reactotron-js`, neither had code changes, nor did the 1 library they rely on `reactotron-core-client`.  

Hence no bump.

You have the latest version, but unfortunately that's not clear.  I kinda picked the wrong option when versioning... i'll think about it some more.

Thanks for filing the issue!

**Comment by tsemerad** [comment]
Turns out my bug was fixed by using `require` rather than `import`. I was attempting to conditionally require and use `Reactotron.createStore` when in dev, but it kept giving me the error. Switching to `import` fixed it. I guess I'll just have it as a regular dependency and just not use it in prod.

**Comment by tsemerad** [comment]
Thanks for the explanation :)
